config.json

```diff
@@ -3,7 +3,11 @@
     "url": "http://engineering.underdog.io",
     "name": "Underdog.io Engineering blog",
     "owner": "Underdog.io Engineering",
-    "description": "Articles and insights from the Underdog.io Engineering team"
+    "description": "Articles and insights from the Underdog.io Engineering team",
+    "gaId": {
+      "production": "UA-49880770-1",
+      "development": "UA-64967323-1"
```

Comment:
One more option is to not define it in locals at all but `env-locals` and then start using `shallow-settings` so we have a `common`, `development`, and `production`